src/helpers/themes.js

```diff
@@ -1,10 +1,18 @@
-import { createMuiTheme } from "@material-ui/core/styles";
+import { createTheme } from '@material-ui/core/styles';
 
-export const basicTheme = createMuiTheme({
-  shadows: ["none"],
+export const basicTheme = createTheme({
+  shadows: ['none'],
   palette: {
     primary: {
-      main: "#455a64",
+      main: '#455a64',
+    },
+  },
+});
+
+export const theme = createTheme({
```

Comment:
your #302 pr has another `theme` variable with exact `palette.primary.main` value in `src/index.js` file. can you check again?

Reply:
I added in #302  to test in the app. I have removed it from that pr. 
#303 has theme setup